src/bda/plone/discount/calculator.py

```diff
@@ -120,6 +134,11 @@ def lookup_cascade(self, context):
             group_lookup = self.group_lookup_factory(
                 context, self.date, group)
             lookups.append(group_lookup)
+        if self.coupon:
+            try:
```

Comment:
Because I have implemented only coupons for CartItems and not also for Cart. I added try just for safety reasons. Should I remove it?